src/compiler/crystal/command.cr

```diff
@@ -118,7 +119,16 @@ class Crystal::Command
     if @config.try(&.output_format) == "json"
       puts ex.to_json
     else
-      puts ex
+      if @show_backtrace
+        puts ex
```

Comment:
No. `inspect_with_backtrace` uses `message`, but full error message of `Crystal::Exception` is built by `to_s` method. So if use `inspect_with_backtrace`, we can't see full error message.